generators/client/files-angular.js

```diff
@@ -126,6 +126,7 @@ const files = {
                 'entities/_entity.module.ts',
                 // home module
                 'home/_index.ts',
+                { file: 'home/_home.module.ts', method: 'copyJs' },
```

Comment:
no need for copyJs as its required only if we have any i18n regex to strip. may be I should rename the method names to be more meaningful